### PR TITLE
Fixes for 0.7

### PR DESCRIPTION
**Pull request by JackDunnNZ**
Tests now pass locally for me on 0.6 and 0.7

### CODE REVIEW

**Comment by AsafManela** [comment]
bump. would be great if this was merged. thanks!

**Comment by AsafManela** [comment]
@JackDunnNZ, perhaps if you changed REQUIRE to julia 0.7 and removed julia 0.6 from .travis.yml, the CI build would pass and someone at @JuliaStats would be kind enough as to merge this

File: REQUIRE [hunk 1]
```diff
@@ -1,4 +1,4 @@
-julia 0.6
+julia 0.7
 Distributions
 StatsBase
-Compat 0.27
+Compat 0.50
```
Comment (ararslan) [inline REQUIRE]: You can actually drop the Compat dependency now that 0.7 is required.

Comment (JackDunnNZ) [reply REQUIRE]: Thanks, good call. Fixed

**Comment by JackDunnNZ** [comment]
Thanks @ararslan!

**Pull request merged**